go.mod

```diff
@@ -26,28 +26,28 @@ require (
 	k8s.io/apimachinery v0.25.4
 	k8s.io/client-go v0.25.4
 	k8s.io/code-generator v0.25.4
-	k8s.io/kube-openapi v0.0.0-20220803162953-67bda5d908f1
-	knative.dev/eventing v0.36.1-0.20230223055929-07e6d0c0c481
-	knative.dev/hack v0.0.0-20230217131352-f2f3107be156
-	knative.dev/pkg v0.0.0-20230222150228-2fdd6bf8be31
+	k8s.io/kube-openapi v0.0.0-20221012153701-172d655c2280
+	knative.dev/eventing v0.37.1-0.20230502055954-cd50d2786189
```

Comment:
Shouldn't this be v0.37.0?

Reply:
We have had other releases that are not from v0.XY.0 so I don't think is that important in this package, we just create patches if something is needed or wrong, not on each Knative Patch